src/main/java/org/craftercms/studio/api/v2/service/dependency/DependencyService.java

```diff
@@ -64,4 +65,13 @@ List<String> getHardDependencies(String site, List<String> paths)
      * @return list of paths dependent on given paths
      */
     List<String> getDependentItems(String siteId, List<String> paths) throws SiteNotFoundException;
```

Comment:
Should we rename this method to prevent it from being confused with the new one?
`getDependentPaths` ?
Same for the the internal service interface

Reply:
I updated this method name to `getDependentPaths`. Thank you

Reply:
I updated that method's name to `getDependentPaths`. Thank you.